dot/rpc/modules/dev_test.go

```diff
@@ -1 +1,326 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
 package modules
+
+import (
+	"errors"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+
+	apimocks "github.com/ChainSafe/gossamer/dot/rpc/modules/mocks"
+)
+
+func Test_uint64ToHex(t *testing.T) {
+	type args struct {
+		input uint64
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "uint64ToHex one",
+			args: args{
+				input: uint64(1),
+			},
+			want: "0x0100000000000000",
+		},
+		{
+			name: "uint64ToHex zero",
+			args: args{
+				input: uint64(0),
+			},
+			want: "0x0000000000000000",
+		},
+		{
+			name: "uint64ToHex max",
+			args: args{
+				input: uint64(18446744073709551615),
+			},
+			want: "0xffffffffffffffff",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := uint64ToHex(tt.args.input); got != tt.want {
+				t.Errorf("uint64ToHex() = %v, want %v", got, tt.want)
+			}
```

Comment:
nit use testify's `assert` lib

```suggestion
			got := uint64ToHex(tt.args.input)
			assert.Equal(t, tt.want, got)
```

Unless it's generated by your IDE then up to you

Reply:
It was generated but I do like assets better I think, so I can work on shifting

Reply:
I also think there were redundant checks here for a lot of these tests, so I think this change will clean it up a bit more as well